### ISSUE TITLE
Description of Kernel_width in lime_image seems to be updated

### ISSUE THREAD

**Issue by remindchobits** [marcotcr/lime#618]
In lime_image kernel width is a constant 0.25 instead of the sort(number of column) * 0.75? Maybe this is only for tabular data..

**Comment by marcotcr** [comment]
Correct, there is an error in the documentation, the default is just 0.25. Do you want to submit a very small pull request? :)
